stl/inc/format

```diff
@@ -3614,6 +3614,50 @@ _NODISCARD size_t formatted_size(const locale& _Loc, const wformat_string<_Types
     return _Buf._Count();
 }
 
+#if _HAS_CXX23
+enum class _Add_newline : bool { _Nope, _Yes };
+
+_NODISCARD inline string _Unescape_braces(const _Add_newline _Add_nl, const string_view _Old_str) {
+    string _Unescaped_str;
+
+    if (_Old_str.empty()) {
+        return _Unescaped_str;

```

Comment:
Missing `_Add_nl` check?

```suggestion
        if (_Add_nl == _Add_newline::_Yes) {
            return _Unescaped_str + '\n';
        } else {
            return _Unescaped_str;
        }
```

Reply:
You were absolutely right, excellent catch. :heart_eyes_cat: I went with a slightly different fix to preserve the NRVO.